recipe/install-lib.sh

```diff
@@ -1,3 +1,29 @@
+mkdir -p $PREFIX/lib $PREFIX/include
+pushd cadical
+    export LDFLAGS="$LDFLAGS -Wl,-headerpad_max_install_names"
+    ./configure
+    make -j8
+    if [ -f build/libcadical.so ]; then
+        cp build/libcadical.so $PREFIX/lib/
```

Comment:
```suggestion
    if [ -f build/libcadical${SHLIB_EXT} ]; then
        cp build/libcadical${SHLIB_EXT} $PREFIX/lib/
```
And you can drop the later test